src/api/app.py

```diff
@@ -205,6 +207,24 @@ def log_out():
 def add_user_event():
     return event_controller.add_event(json.loads(request.data))
 
+@app.route('/api/addcourse', methods=['POST'])
```

Comment:
conforming with the recent meeting, would be nice if you could change the endpoint names to be `/api/course` and then just have the method vary (e.g. POST would be equivalent to your old '/api/addcourse', etc.)

Reply:
Done